CONTRIBUTING.md

```diff
@@ -0,0 +1,156 @@
+# Contributing to `dbt-trino`
+
+## Getting the code
+
+### Installing git
+
+You will need `git` in order to download and modify the `dbt-core` source code. On macOS, the best way to download git is to just install [Xcode](https://developer.apple.com/support/xcode/).
+
+### How to contribute?
+
+If you are not a member of the `starburstdata` GitHub organization, you can contribute to `dbt-trino` by forking the `dbt-trino` repository. For a detailed overview on forking, check out the [GitHub docs on forking](https://help.github.com/en/articles/fork-a-repo). In short, you will need to:
```

Comment:
Let's leave `starburstdata` out of the picture. The guideline is relevant mainly for the open source community where the members will likely need to do forks.